Kickstarter-iOS/AppDelegate.swift

```diff
@@ -15,6 +15,31 @@ import ReactiveSwift
 import Result
 import SafariServices
 import UIKit
+import UserNotifications
+
+@available(iOS 10.0, *)
+extension UNAuthorizationStatus: NotificationAuthorizationStatusType {
+  public var isAuthorized: Bool {
+    switch self {
+    case .authorized: return true
+    case .denied, .notDetermined: return false
+    }
+  }
+
+  public var isDenied: Bool {
+    switch self {
+    case .denied: return true
+    case .authorized, .notDetermined: return false
+    }
+  }
+
+  public var isNotDetermined: Bool {
+    switch self {
+    case .notDetermined: return true
+    case .denied, .authorized: return false
+    }
+  }
+}
```

Comment:
This is work around of the issue that `UNAuthorizationStatus` was introduced in iOS 10.